book/plutocracy_report.ipynb

```diff
@@ -239,12 +116,13 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Here we can see that for the DAOs which we choose to analyze the top 5% of voters have a clear, disproportionate amount of voting power compared to the average voter.\n",
+    "For the DAOs above, the top 5% of voters have a disproportionate amount of voting power compared to the average voter.\n",
     "\n",
-    "We can also gain some insight into the economic might that these whales hold for each DAO at the time of voting by taking a look at the average cost of each vote for each DAO's native token at the time these proposals were active (in USD)."
+    "We also have an insight into the economic might whales hold in their respective DAO. We calculate the average cost to purchase a token to vote at the time the proposal was active (in USD) for comparative purposes."
```

Comment:
Discussed this with Alex, and we agree that calculating the "cost of vote" metric is tricky and easy to pick holes in. Explaining that we're using the metric for comparison alone reduces the scrutiny it may face.